### PR TITLE
chore: Add kn-service-log to peribolos configuration

### PR DESCRIPTION
**Pull request by rhuss**
(and sort the kn-plugins alphabetically)

### CODE REVIEW

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

File: peribolos/knative-sandbox.yaml [hunk 1]
```diff
@@ -365,7 +365,7 @@ orgs:
       kn-plugin-diag:
         allow_merge_commit: false
         allow_rebase_merge: false
-        description: Kn plugin for exposing detailed information for different layers of knative objects for diagnose.
+        description: Kn plugin for exposing detailed information for different layer of knative objects for diagnose.
```
Comment (markusthoemmes) [inline peribolos/knative-sandbox.yaml:368]: bad merge I guess?

Comment (rhuss) [reply peribolos/knative-sandbox.yaml]: true dat. thx!

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Pull request merged**